src/app/store/technical-records/actions/technical-record-service.actions.ts

```diff
@@ -52,6 +52,10 @@ export const updateEditingTechRecord = createAction(`${prefix} updateEditingTech
 export const updateEditingTechRecordCancel = createAction(`${prefix} updateEditingTechRecordCancel`);
 
 export const changeVehicleType = createAction(`${prefix} changeVehicleType`, props<{ vehicleType: VehicleTypes }>());
+//not sure on this
+export const amendVin = createAction(`${prefix} amendVin`, props<{ oldVin: string; newVin: string; systemNumber: string }>());
+export const amendVinSuccess = createAction(`amendVin Success`);
+export const amendVinFailure = createOutcomeAction(`amendVin`);
```

Comment:
Not super important, but you don't need the backticks if you're not doing interpolation